### ISSUE TITLE
Request: filter by party member name

### ISSUE THREAD

**Issue by rideekulous** [Vloer/keycount#10]
Seems that `/kcf player {CHARACTER_NAME}` only applies to the user's characters, is that intended? It would be nice to be able to do something like `/kcf member {CHARACTER_NAME}` to quickly print out data for a character you previously ran a M+ dungeon with.

E.g.,
The command `/kcf player Swiftmend` outputs nothing.
![image](https://github.com/Vloer/keycount/assets/165203372/d877f35c-7dea-4112-83cd-954240200ee1)

... but if I search using the UI (Player lookup), data is displayed.
![image](https://github.com/Vloer/keycount/assets/165203372/79922954-7a85-414e-8446-22f86bd37ab5)

**Comment by Vloer** [comment]
Yes, this behaviour is intended. I'll write a quick implementation somewhere this week that allows you to get the success stats (first tab in player lookup view) in your chat

**Comment by Vloer** [comment]
Added in latest release

**Comment by rideekulous** [comment]
I appreciate you adding the summary feature, but I was hoping the data would mimic what `/kcf player {CHARACTER_NAME}` does. Is the output configurable?

**Comment by Vloer** [comment]
There is no settings screen at the moment so I changed /kcp to include dungeon runs, and added /kcs for the previous functionality in [the latest release](https://github.com/Vloer/keycount/releases/tag/v0.7.4-beta)

**Comment by rideekulous** [comment]
Very cool, much appreciated!
